package.json

```diff
@@ -18,12 +18,14 @@
     "pretest": "npm run lint",
     "prewatch:browser": "npm run clean:browser",
     "prewatch:mobile": "npm run clean:mobile",
-    "lint": "standard 'src/**/*.js' 'src/**/*.jsx' 'mobile/src/**/*.js' 'mobile/src/**/*.jsx'",
+    "lint": "npm-run-all --parallel 'lint:*'",
+    "lint:styles": "stylint src/styles --config ./.stylintrc",
+    "lint:js": "standard 'src/**/*.js' 'src/**/*.jsx' 'test/**/*.js' 'mobile/src/**/*.js' 'mobile/src/**/*.jsx' 'mobile/test/**/*.js'",
     "test": "node --harmony_proxies node_modules/.bin/jest",
     "watch": "webpack --config webpack.config.js --watch --display-modules --display-chunks",
```

Comment:
`npm run watch` does not work anymore since it does not define a `NODE_ENV`:

```
/Users/pierre/code/cozy-files-v3/webpack.config.js:6
const target = process.env.NODE_ENV.match(/^(\w+):/)[1]
                                   ^

TypeError: Cannot read property 'match' of undefined
```